frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -2016,6 +2016,7 @@ export async function getTimeTrackingForUserSummedPerAnnotation(
   if (annotationTypes != null) params.append("annotationTypes", annotationTypes);
   if (projectIds != null && projectIds.length > 0)
     params.append("projectIds", projectIds.join(","));
+  params.append("annotationStates", "Active,Finished");
```

Comment:
This is just a prerequisite to show the annotations state in the table and in the CSV, and to filter it, right?

Reply:
Right, only for the filtering at this time. I wrote this before I learned that the priority is lower for now. I guess we can use it in the frontend in a later iteration, or remove it again if we decide we don’t want to go for this option :)